vars/RedHat.yml

```diff
@@ -0,0 +1,7 @@
+---
+pulp_preq_packages:
+  - python3
+  - python3-devel
+  - libselinux-python  # For ansible itself
+
+pulp_psycopg_package: 'python{{ ansible_python.version.major }}-psycopg2'
```

Comment:
Pulp 3 requires Python 3. Shouldn't python3-psycopg2 always be installed?

Reply:
ah no, we use python3 in a venv. Ansible uses whatever python it can find, so we need the psycopg for that specific version.

If needed, we can add some more magic for other OS releases/versions.

Reply:
Ansible can be told which version of Python to use with `ansible_python_version` in a playbook:

```yaml
- hosts: all
  vars:
    ansible_python_version: /usr/bin/python3
  roles: "..."
```

We do so already. [Link.](https://github.com/pulp/devel/blob/3.0-dev/ansible/deploy-pulp3.yml#L6). Does that make a difference?

Reply:
Not all users that import this role into their existing playbooks will have this though.

Reply:
Roles can define requirements, and enforce them through assertions. I'm guessing your response is "I want to support Python 2 for Ansible?"

Reply:
If you say you absolutely want this, I can type it in, but I prefer to have more compatibility and as far as I can see, nothing will break if we use python 2 or 3 for Ansible and python 3 for the Pulp environment.

As a datapoint [Ansible is not fully supported](http://docs.ansible.com/ansible/latest/python_3_support.html) on Python 3 yet

Reply:
> If you say you absolutely want this

No, I'm not set on using Python 3 for Ansible. I'm trying to find out the rationale for the current design choice. Wider compatibility is a good rationale.